test_conformance/subgroups/subhelpers.h

```diff
@@ -22,6 +22,16 @@
 
 #include <limits>
 #include <vector>
+#include <type_traits>
+
+#undef min
+#undef max
+
+// Adjust these individually below if desired/needed
+#define G 2000
+#define L 200
+
+#define NON_UNIFORM 4
```

Comment:
These should definitely be proper constants in a relevant namespace, correctly documented.

There is no indication as to what they mean.

Reply:
Is this ok ?
`#define NON_UNIFORM_WG_SIZE 4`